data/tlg0020/tlg002/tlg0020.tlg002.perseus-grc2.xml

```diff
@@ -82,7 +90,7 @@
     <change when="2016-02-18" who="TDBuck">converted to CTS EpiDoc</change>
 </revisionDesc>
 </teiHeader>
-<text xml:id="stoa0045.stoa0">
+    <text  xml:lang="grc">
```

Comment:
We just use `<text>` language is invoked in the first div

Reply:
I figured this was just a typo but I standardized it to tlg0020.tlg001.

Reply:
Yes, that is no longer part of the header. Just `<text>`